src/api.rs

```diff
@@ -1123,6 +1126,12 @@ impl<T: Pixel> ContextInner<T> {
     if let Some((last_frame, _)) = frames_to_process.iter().last() {
       self.next_lookahead_frame = last_frame + 1;
     }
+
+    // Compute the frame invariants.
```

Comment:
This comment is no longer accurate?

Reply:
Wdym? This is the lookahead code that computes the frame invariants.

Reply:
ah whoops was reading wrong line